explorer/file_test.cpp

```diff
@@ -17,7 +18,12 @@ namespace {
 
 class ExplorerFileTest : public FileTestBase {
  public:
-  using FileTestBase::FileTestBase;
+  explicit ExplorerFileTest(std::filesystem::path path)
+      : FileTestBase(std::move(path)),
+        prelude_line_re_(R"(prelude.carbon:(\d+))"),
+        timing_re_(R"((Time elapsed in \w+: )\d+(ms))") {
+    CARBON_CHECK(prelude_line_re_.ok()) << prelude_line_re_.error();
```

Comment:
Why do we check one of these but not the other?

Reply:
Added a check for the other. Just what I'd run into mistakes with. :)